### PR TITLE
{compiler}[dummy] GCCcore 5.4.0 + GCC 5.4.0-2.26 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 7 (7 easyconfigs in this PR)
node2564.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0bb88a88686ecb446c832974ceb267e5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 7 (7 easyconfigs in this PR)
node2137.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/55be9bc857446d4fdcc630506420c76c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 7 (7 easyconfigs in this PR)
node2575.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/27387b2e31d5ed54de168780af2ef284 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 7 (7 easyconfigs in this PR)
node2013.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/002ca31a578aeec8a629f2790a8e6a8f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2575.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1cb0f7bdfb46de6dbf8bc70fda048fd2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2013.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/862d16e2f5631349cafc29310993db75 for a full test report.

**Comment by boegel** [comment]
@wpoely86 please review

File: easybuild/easyconfigs/g/GCCcore/GCCcore-5.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+sources = [
+    'gcc-%(version)s.tar.bz2',
+    'gmp-6.1.0.tar.bz2',
+    'mpfr-%s.tar.gz' % mpfr_version,
+    'mpc-1.0.3.tar.gz',
+    'isl-0.15.tar.bz2',
+]
+
+patches = [('mpfr-%s-allpatches-20150603.patch' % mpfr_version, '../mpfr-%s' % mpfr_version)]
```
Comment (wpoely86) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-5.4.0.eb]: use the latest one (May I think).

File: easybuild/easyconfigs/g/GCCcore/mpfr-3.1.4-allpatches-20150603.patch [hunk 2]
```diff
@@ -0,0 +1,356 @@
+diff -Naurd mpfr-3.1.4-a/PATCHES mpfr-3.1.4-b/PATCHES
```
Comment (wpoely86) [inline easybuild/easyconfigs/g/GCCcore/mpfr-3.1.4-allpatches-20150603.patch]: add original url

File: easybuild/easyconfigs/g/GCCcore/GCCcore-5.4.0.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-5.4.0.eb]: I screwed up the date stamp, this patch is the one I grabbed on June 3rd 2016, so it should be `20160603`; or are you using another way of versioning the patch?

**Comment by boegel** [comment]
@wpoely86 datestamp in MPFR patch fixed, I used the last changed date

**Comment by wpoely86** [comment]
:+1:

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
node2447.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7102c891d3d503f6730212c42acece0f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
node2069.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/0aaab47926049adbcde1cf40aedb6c48 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2069.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ef4e7f89f48df0ee8197888a63e4090f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2515.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/65b3650163fb3646b64c63db3e55c386 for a full test report.

**Comment by boegel** [comment]
Thanks for the feedback @wpoely86!

**Pull request merged**